test/mixtures.jl

```diff
@@ -340,12 +356,15 @@ end
     @test string(to^2) == " ( 1.0 + 𝒪(t⁴)) t² + 𝒪(t¹⁰)"
     @test ti + to == Taylor1([ti, one(ti)], 9)
     tito = ti * to
+    # @test ti > ti^2 > to > 0
```

Comment:
I understand (very well) that you want to keep the question of these orderings out of scope of this PR. Can you create a separate issue for them and link to that issue in the comments? (Both in the tests and in the code.) Commented code is a bit of a smell, but I do agree that it would be a shame to delete this altogether.

Reply:
Thanks for the suggestion; I opened #326, and include it in the comments